steps/generate_analytical_dataset.py

```diff
@@ -16,6 +17,12 @@
 from datetime import datetime
 import logging
 import pytz
+from logger import setup_logging
+
+level = "info"
+logger_path = "/var/log/adg/generate_analytical_dataset_log.log"
+logger_format = "{ 'timestamp': '%(asctime)s', 'log_level': '%(levelname)s', 'message': '%(message)s' }"
+the_logger = setup_logging(level, logger_path , logger_format)
```

Comment:
I don't think this is the best way to do the logging. You've ended up with the logging level hard-coded, which would now be applied to every logging call and isn't super useful.

A better approach might be something like:

```
import logging
logging.basicConfig(filename='/var/log/adg/generate_analytical_dataset_log',level=logging.INFO)
```

I appreciate you are currently overriding the level with the `the_logger.info` call later, but you should only need to override with `debug` calls if you're setting your standard to be `info`.

To go a step further you could potentially reference an existing python-based logging implementation such as here: https://github.ucds.io/dip/aws-internal-compute/blob/master/lambdas/uc_export_to_crown_controller/src/uc_ecc_lambda/uc_export_to_crown_controller.py#L32 which would allow you to conform to our slightly more robust standards.

Reply:
we are already using a form of the logger linked, i think it was implemented earlier. I just cleaned it up a little bit.